solr/core/src/java/org/apache/solr/filestore/DistribFileStore.java

```diff
@@ -634,7 +635,7 @@ public Map<String, byte[]> getKeys() throws IOException {
   // reads local keys file
   private static Map<String, byte[]> _getKeys(Path solrHome) throws IOException {
     Map<String, byte[]> result = new HashMap<>();
-    Path keysDir = _getRealPath(FileStoreAPI.KEYS_DIR, solrHome);
+    Path keysDir = _getRealPath(ClusterFileStore.KEYS_DIR, solrHome);
```

Comment:
Sigh I hate you `_` methods and objects!

Reply:
Agreed, I don't love the convention either.  There's several of these in DistribFileStore though - enough that I'm worried renaming will bloat this PR too much 😦

Reply:
Was it "errorprone" that we used to help us clean up our code and enforce some conventions?  For another JIRA, see if errorprone would flag any "_" methods for example.   We *do* need to make all our conventions enforced via automated check, there are too many people over too many years to otherwise keep code standards intact...